matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/read/ReadService.kt

```diff
@@ -63,6 +63,14 @@ interface ReadService {
      */
     fun getMyReadReceiptLive(): LiveData<Optional<String>>
 
+    /**
+     * Get the eventId where the read receipt for the provided user is
+     * @param otherUserId the userId param to look for
+     *
+     * @return the eventId where the read receipt for the provided user is attached, or null if not found
+     */
+    fun getUserReadReceipt(otherUserId: String): String?
```

Comment:
Why are we using "otherUserId" term instead of "userId"?

Reply:
Because in the implementation we already have a `userId`. But you are right, I will change to `userId`, it's better